cookbook/product/manipulate-product.rst

```diff
@@ -0,0 +1,12 @@
+How to Manipulate the Products
```

Comment:
How to manipulate products (manipulate the product will mean you know which product you want to manipulate)